src/components/empty-list/empty-list.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+  Title,
+  Button,
+} from "@patternfly/react-core";
+import { PlusCircleIcon } from "@patternfly/react-icons";
+
+type EmptyListProps = {
+  message: string;
+  instructions: string;
+  primaryActionText: string;
+  primaryAction: any;
```

Comment:
We want to avoid using `any`
```suggestion
  primaryAction: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Reply:
Also I would recommend to name actions "on(action name)" so in this case `onPrimaryAction` that way you can also try them out in storybook
```suggestion
  onPrimaryAction: MouseEventHandler<HTMLButtonElement>;
```